### DISCUSSION TITLE
Can I make my pump not turn off?

### DISCUSSION THREAD

**Discussion by NapTight** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#542] [Q&A]
Front the manual:

Can I make my pump not turn off? Yes, on the Config.html page, add a command in the commandqueue to be repeated daily. E.g. Set pump 1 every 86400 s 

Is there a way to do this from the GUI? I see the files in there, can they be modified, or do I need to modify and recompile with a computer?

**Comment by visualapproach** [comment]
![image](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/22987250/2e922d8e-6387-4e20-97f9-caaafe7b2b9c)

**Reply by NapTight** [reply]
Yeehaa! 
 
This is you, ya?
Thomas Landahl
@TLandahl

**Reply by visualapproach** [reply]
Yup

**Reply by NapTight** [reply]
Thanks for putting this together

**Reply by visualapproach** [reply]
You are welcome! Thanks for contributing!
